src/actors/symbolication.rs

```diff
@@ -192,19 +192,21 @@ impl SymbolicationActor {
 
             let os_name = state.system_info().os_name();
 
-            Ok((
-                byteview,
-                state
-                    .referenced_modules()
-                    .into_iter()
-                    .filter_map(|code_module| {
-                        Some((
-                            code_module.id()?,
-                            object_info_from_minidump_module(&os_name, code_module),
-                        ))
-                    })
-                    .collect::<Vec<_>>(),
-            ))
+            let referenced_modules = state.referenced_modules();
+
+            let object_infos: Vec<_> = state
+                .referenced_modules()
```

Comment:
Should this call `state.modules()` instead, so that it contains all modules? 

Also, I would suggest rebuilding this list from the second process state. Due to the use of CFI, the list of referenced modules might change slightly (a module might no longer be referenced since it was a false-positive in the stack scan).

Reply:
Yes, it should be `modules()`. This list will be discarded before calling out to symbolication so the "unused" status should be correct.

I initially thought that we will later change this to use these object infos as a base for the debug images response but I am reconsidering this now.

Reply:
this should be fixed now